community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java

```diff
@@ -141,6 +144,11 @@ private ThirdPartyJaxRsPackage createThirdPartyJaxRsPackage( String packageAndMo
     @Description( "Enable HTTP request logging." )
     public static final Setting<Boolean> http_logging_enabled = setting( "dbms.logs.http.enabled", BOOLEAN, FALSE );
 
+    @Description( "Path to HTTP request log." )
+    public static final Setting<File> http_log_path =
+            derivedSetting( "dbms.logs.http.path", logs_directory, ( logs ) -> new File( logs, "http.log" ),
+                    PATH );
+
```

Comment:
It is not super important, I simply find confusing using the same term for 2 different things.